### PR TITLE
database seeding

### LINKED ISSUE #273: Database Seed
Seed the following data to the database. We already have administrator@circuitverse.org. Keep all passwords  as "password". The seeds can be done in separate PRs if necessary. 

- [ ] user1 - user1@circuitverse.org, user2 -  user2@circuitverse.org
- [ ] Make user1 a group mentor of group "group1", add user 2 to the group

Add the example circuits to user1 - https://circuitverse.org/examples

- [ ] SAP-1
- [ ] Full Adder
- [ ] ALU 74LS181
- [ ] Add some more interesting ones of your own choice
- [ ] Seed 3 circuits names "public", "private" and "Limited Acess", the circuits should have the corresponding access type respectively 

**Are you working on this? (Yes/No)**

No

### PR DESCRIPTION
**Pull request by shubhankarsharma00**
Fixes #273 

#### Describe the changes you have made in this pr -
added user1 and user2 along with a group in which user1 is mentor and user2 is member.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -11,3 +11,31 @@
...
...,"id":"837tujwUlLFHp8nlQgTN"}]}},{"x":340,"y":310,"objectType":"Input","label":"","direction":"RIGHT","labelDirection":"LEFT","propagationDelay":0,"customData":{"nodes":{"output1":6},"values":{"state":0},"constructorParamaters":["RIGHT",1,{"x":0,"y":60,"id":"PuthdFyas7NQ66FMFOqd"}]}}],"Output":[{"x":670,"y":310,"objectType":"Output","label":"","direction":"LEFT","labelDirection":"RIGHT","propagationDelay":0,"customData":{"nodes":{"inp1":16},"constructorParamaters":["LEFT",1,{"x":100,"y":20,"id":"Af0TIhW3aLK88scEqtqf"}]}},{"x":720,"y":240,"objectType":"Output","label":"","direction":"LEFT","labelDirection":"RIGHT","propagationDelay":0,"customData":{"nodes":{"inp1":17},"constructorParamaters":["LEFT",1,{"x":100,"y":40,"id":"fVe0eexwXF7A8QKIC60t"}]}}],"XorGate":[{"x":670,"y":240,"objectType":"XorGate","label":"","direction":"RIGHT","labelDirection":"LEFT","propagationDelay":100,"customData":{"constructorParamaters":["RIGHT",2,1],"nodes":{"inp":[13,14],"output1":15}}}],"SubCircuit":[{"x":390,"y":210,"id":"38912139787","inputNodes":[2,3],"outputNodes":[0,1],"version":"2.0"},{"x":510,"y":270,"id":"38912139787","inputNodes":[9,10],"outputNodes":[7,8],"version":"2.0"}],"nodes":[11,12]}]}',
```
Comment (satu0king) [inline db/seeds.rb]: I don't think it is a good idea to have the json data inline with the code. Can you have this in a json file fullAdder.json and read it during runtime? @tachyons might have some inputs

Comment (shubhankarsharma00) [reply db/seeds.rb]: ok

Comment (tachyons) [reply db/seeds.rb]: Yeah, loading from a json file will be much better

**Comment by shubhankarsharma00** [comment]
@satu0king the json is now read at the run time added SAT and ALU-74LS181. Also added tags to Public, Private and Limited access projects.

**Comment by satu0king** [comment]
Looks good to me.

**Pull request merged**